packages.ini

```diff
@@ -346,6 +346,7 @@ custom_prebuild = prebuild/crc32c 1.1.2
 [kubernetes==24.2.0]
 
 [kubernetes-stubs==22.6.0.post1]
```

Comment:
hmmm now that I look at this package it's missing a `LICENSE` so we probably can't use it :(